autoload/goyo.vim

```diff
@@ -33,8 +33,12 @@ function! s:get_color(group, attr)
 endfunction
 
 function! s:set_color(group, attr, color)
-  let gui = a:color =~ '^#'
-  execute printf('hi %s %s%s=%s', a:group, gui ? 'gui' : 'cterm', a:attr, a:color)
+  if empty(a:attr)
```

Comment:
How about if we change this to `if a:color == 'NONE'`? Since there is `call s:set_color(grp, 'bg', 'NONE')` as well.